Client/Frontend/Browser/TabManager.swift

```diff
@@ -736,13 +736,12 @@ extension TabManager {
                 }
             )
         }
+        isRestoring = false
```

Comment:
I moved this up here because otherwise the tab that is added at line 742 never gets events triggered preventing TopTabs from animating it.